### PR TITLE
new capabilities for more timeouts and way of changing the way we find elements

### PR DESCRIPTION
**Pull request by Jared1048**
NewCommand timeout
idleTimeout

We found a bug/feature in Appium that was making it hard to find an element so for some of our tests we need to add change this with the new cap. of simpleIsVisibleCheck (Appium has more "smart" logic to help with web element validations which we don't need)

### CODE REVIEW

**Comment by konwinkler** [comment]
the continuous integration is failing because there are tests which check the capabilities being built. After those are passing I see no reason to not include this.

File: src/test/resources/test_yaml/android_full.yaml [hunk 1]
```diff
@@ -16,3 +17,9 @@ defaults:
     appActivity: LaunchActivity
     appWaitActivity: HomeActivity
     intentCategory: android.intent.category.LEANBACK_LAUNCHER
+    newCommandTimeout: 600
+    idleTimeout: 600
```
Comment (jossjacobo) [inline src/test/resources/test_yaml/android_full.yaml]: What is the default value if these are not included in the config?

Comment (Jared1048) [reply src/test/resources/test_yaml/android_full.yaml]: Im still working through the details of what these should be set at . Im currently at values 500 and 270 respectively. Id like the defaults to be nothing and only use them if specified in the yaml file

Comment (jossjacobo) [reply src/test/resources/test_yaml/android_full.yaml]: kk, we should test that scenario, remove either `newCommandTimeout` or `idleTimeout` from the configs and verify that it's passing empty or nothing to the `desiredCapabilities`.

File: src/main/java/com/joss/conductor/mobile/Locomotive.java [hunk 2]
```diff
@@ -529,7 +532,7 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
     private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection direction, float percentage, int duration) {
         Dimension screen = getAppiumDriver().manage().window().getSize();
 
-         final int SCREEN_MARGIN = 10;
+         final int SCREEN_MARGIN = 130;
```
Comment (konwinkler) [inline src/main/java/com/joss/conductor/mobile/Locomotive.java]: Changing the margin seems unrelated to the original intent of this PR. A better solution than increasing it for everyone would also be to overload `performCornerSwipe()` with an additional parameter to overwrite the default screen margin.
Could we handle this in a separate PR?

Comment (Jared1048) [reply src/main/java/com/joss/conductor/mobile/Locomotive.java]: yeah, this wasn't meant to be in this pr
